docs/getting-started-guides/docker.md

```diff
@@ -162,14 +162,22 @@ See [here](/docs/getting-started-guides/docker-multinode/deployDNS/) for instruc
 
 ### Turning down your cluster
 
-1. Delete all the containers including the kubelet:
+1\. Delete the nginx service and deployment:
```

Comment:
No need for backslash before the period. Delete this sentence since this step is optional.